examples/components-macos/lib/Tabs.re

```diff
@@ -0,0 +1,115 @@
+open Brisk_macos;
+
+let hairline = (~style=[], ~children as _: list(unit), ()) => {
```

Comment:
I don't think `~_children` is a valid syntax.
Type annotation is unfortunately necessary, this could be streamlined with PPX in future or even language changes.

Reply:
This seems to be valid: `let f = (~_x, ()) => 1;`. I'm curious about this, will take a closer look at the repo soon.